### PR TITLE
Explore UI

### PR DESCRIPTION
**Pull request by divyanshgandhilm**
Only to be merged after "Profile UI" merge

### CODE REVIEW

File: lib/views/chatroom/views/chatroom_participants_page.dart [hunk 1]
```diff
@@ -79,67 +82,55 @@ class ParticipantItem extends StatelessWidget {
...
             ),
           ),
-          const SizedBox(width: 16),
-          Text(
-            name,
-            style: GoogleFonts.montserrat(
-              fontSize: 16,
-              fontWeight: FontWeight.w600,
+        );
+        Navigator.push(context, route);
```
Comment (PaRaDoX50) [inline lib/views/chatroom/views/chatroom_participants_page.dart:96]: I guess named routing is a better option.
While navigation without using named routes is OK for smaller projects, in more complex apps it adds code duplication.

File: lib/views/explore/explore_page.dart [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                                _spaces = Spaces.mostMessages;
+                              });
+                              Navigator.pop(context);
+                            },
+                          ),
+                        ],
+                      ),
+                    );
+                  },
+                );
```
Comment (PaRaDoX50) [inline lib/views/explore/explore_page.dart]: it will be better to separate this modalsheet code to a different function

**Review by PaRaDoX50 — CHANGES_REQUESTED** [review]

File: lib/views/chatroom/views/chatroom_participants_page.dart (hunk 1 above)
Comment (divyanshgandhilm) [reply lib/views/chatroom/views/chatroom_participants_page.dart:96]: I will be implementing all the routes using the `go_router` package as soon as we start making this more and more complex, the job right now is to nail the UI part the best we can. I get your point, but it is anyway going to be added.

File: lib/views/explore/explore_page.dart (hunk 2 above)
Comment (divyanshgandhilm) [reply lib/views/explore/explore_page.dart]: Good point, done!

File: lib/views/chatroom/views/chatroom_participants_page.dart (hunk 1 above)
Comment (PaRaDoX50) [reply lib/views/chatroom/views/chatroom_participants_page.dart:96]: Okay, but I guess there's no need to implement routing using a package, the primary navigator has enough options to satisfy app navigation.

Comment (PaRaDoX50) [reply lib/views/chatroom/views/chatroom_participants_page.dart:96]: Okay, but I guess there's no need to implement routing using a package, the primary navigator has enough options to satisfy app navigation.

**Review by PaRaDoX50 — APPROVED** [review]

**Pull request merged**